package exceptions

class UserNotFoundException : RuntimeException("User not found!") {
}